internal/app/backend/backend_service.go

```diff
@@ -142,10 +147,67 @@ func (s *backendService) getGRPCClient(config *pb.FunctionConfig_Grpc) (pb.Match
 	return client.(pb.MatchFunctionClient), nil
 }
 
+// nolint: unused
+func (s *backendService) matchesFromHTTPMMF(profile *pb.MatchProfile, client *http.Client, baseURL string, matchChan chan<- *pb.Match, errChan chan<- error) {
+	marshaler := jsonpb.Marshaler{}
+	jsonProfile, err := marshaler.MarshalToString(profile)
+	if err != nil {
+		errChan <- err
+		logger.WithError(err).Error("failed to marshal profile pb to string")
+		return
+	}
+
+	reqBody, err := json.Marshal(map[string]string{"profile": jsonProfile})
+	if err != nil {
+		errChan <- err
+		logger.WithError(err).Error("failed to marshal request body")
+		return
+	}
+
+	req, err := http.NewRequest("post", baseURL+"/v1/matchfunction:run", bytes.NewBuffer(reqBody))
+	if err != nil {
+		errChan <- err
+		logger.WithError(err).Error("failed to create mmf http request")
+		return
+	}
+
+	resp, err := client.Do(req)
+	if err != nil {
+		errChan <- err
+		logger.WithError(err).Error("failed to get response from mmf run")
```

Comment:
There's 2 issues. I'm not sure if these log statements are useful here because they get logged on the RPC responses. Also the amount of information from these log entries does not provide enough information.
Who did I call?
Is there any payload information that would be relevant to the error as well?